index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
+These can be accessed at the following urls:
+
+- Summit: https://summit-lsp.lsst.codes/obsenv-management/dashboard
+- BTS: https://base-lsp.lsst.codes/obsenv-management/dashboard
+
+.. _traceability:
+
+Traceability
+------------
+
+One of the key features of the ``manage_obs_env`` application is that it allows us to trace the user actions and the state of the observing environment.
+Whenever a user runs the ``manage_obs_env`` it will publish the action performed, alongside its parameters, time and the user name of who performed it.
+It also published a summary of the repositories version after the action is performed.
+
+The information is readily available in sasquatch (:sqr:`068`) under the ``obsenv`` database, and can be inspected using Chronograf.
+
+Observing Environment Repositories
+----------------------------------
+
+At the time of this writting the following repositories are part of the observing environment:
+
+- ts_observatory_control
+- atmospec
+- spectractor
+- summit_extras
+- summit_utils
+- ts_externalscripts
+- ts_observing_utilities
+- ts_standardscripts
+- ts_auxtel_standardscripts
+- ts_maintel_standardscripts
+- ts_wep
+- ts_config_ocs
+- ts_config_attcs
+- ts_config_mttcs
+
+
+Systems Configured to use the Observing Environment
+---------------------------------------------------
+
+The following systems are currently setup to use the observing environment.
+
+ScriptQueue
+^^^^^^^^^^^
+SAL Scripts launched by the queue will be from the observing environment repositories.
+
+These includes mainly, ts_observatory_control, ts_standardscripts, ts_maintel_standardscripts, ts_auxtel_standardscripts and ts_externalscripts.
+Having these as part of the observing environment allows us to push updates and hot fixes on the fly.
+Once a package is updated, any SAL Script launched after the update will use the updated version.
+There is no need to cycle the ScriptQueue state.
+However, if there is a change in SAL Script configuration schema or the change is for a new SAL Script, users might have to reload the schema or command to queue to publish the schema.
+
+Reloading can be done directly from LOVE launch script pop up view.
+
+.. image:: /_static/love_reload_schema.png
+   :target: ../_images/love_reload_schema.png
+   :alt: Reload script configuration schema from LOVE launch script pop up view.
+
+For new SAL Scripts, use the ``run_command`` script to send the ``showSchema`` command to the ScriptQueue so it will publish the new script configuration.
+The configuration for the ``run_command`` script will be something like this:
+
+.. code-block:: yaml
+
+  component: ScriptQueue:1 # 1 for MTQueue, 2 for ATQueue and so on.
+  cmd: showSchema
+  parameters:
+    isStandard: true/false
+    path: <path-to-the-new-script>
+
+Nublado
+^^^^^^^
+
+For user's Jupyter notebook to have access to the same versions of the control system packages as those used by the ScriptQueue.
```

Comment:
Could link to obsenv setup doc here https://obs-ops.lsst.io/Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.html